### PR TITLE
57875 step indicator

### PR DESCRIPTION
**Pull request by harshilmoradia**
## Summary

This pull request adds V1 and V3 step indicator in ds playground.

## Related issue(s)
department-of-veterans-affairs/va.gov-team#57875

## Testing done
Manually in Chrome/Edge/Firefox

## Screenshots
Desktop:
Screenshot 2023-05-23 at 5 01 20 PM

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

## Acceptance criteria
- [ ] The V1-compatible and V3-compatible versions of this component render correctly and function correctly when loaded simultaneously within the DS Playground app.

### CODE REVIEW

File: src/applications/ds-playground/pages/V1V3Page.jsx [hunk 1]
```diff
@@ -492,6 +493,34 @@ export default function V1V3Page() {
...
+        {/* Progress bar - segmented */}
+        <div className="vads-l-row">
+          <h3>Progress bar - segmented</h3>
+          <div className="vads-l-col--12 vads-u-align-items--center vads-u-border-bottom--1px vads-u-border-color--primary medium-screen:vads-u-display--flex">
+            <div className="vads-l-col--12 small-screen:vads-l-col--6 vads-u-margin--1">
+              <VaSegmentedProgressBar
+                current={2}
+                heading-text="V1 VA Benefits"
+                total={5}
+                uswds
```
Comment (Andrew565) [inline src/applications/ds-playground/pages/V1V3Page.jsx:506]: Should the V1 progress bar not have the uswds flag?

**Review by Andrew565 — CHANGES_REQUESTED** [review]

File: src/applications/ds-playground/pages/V1V3Page.jsx (hunk 1 above)
Comment (harshil1793) [reply src/applications/ds-playground/pages/V1V3Page.jsx:506]: right, my bad

**Comment by powellkerry** [comment]
@Andrew565 @harshil1793 Should the playground show all the variations for this component (different alignment of the labels, numbers, no numbers, small numbers, labels, no labels) or should that just be in storybook?

**Comment by harshil1793** [comment]
I believe I've addressed all the properties in one example, which should be useful. Any additional details can be found in the Storybook.

**Review by Andrew565 — APPROVED** [review]
LGTM

**Pull request merged**